libkbfs/tlf_journal_test.go

```diff
@@ -1052,7 +1052,7 @@ func testTLFJournalResolveBranch(t *testing.T, ver MetadataVer) {
 	require.Equal(t, bids[0], blocks.puts.blockStates[0].blockPtr.ID)
 	require.Equal(t, bids[2], blocks.puts.blockStates[1].blockPtr.ID)
 
-	// resolveBranch resumes background work.
+	tlfJournal.resumeBackgroundWork()
```

Comment:
It was because the test starts off in paused mode, so resolving the branch isn't enough to resume on its own.  But as a result of the changes I moved things around a bit in this test, so it should be more clear now.